cmd/manager/main.go

```diff
@@ -92,16 +93,31 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := monitoringv1.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "error registering prometheus monitoring objects")
+		os.Exit(1)
```

Comment:
Do we not want to kill the operator?

Reply:
I copied what was above in 93, which is generated by the sdk. Should we follow their pattern or remove?

Reply:
@tparikh whats your opinion here? If we can't register the scheme for the monitoring objects we'll kill the operator.

Reply:
Fwiw, this is in the approved certman-operator pr

Reply:
Ok we'll leave this for another discussion.